great-idea/index.html

```diff
@@ -13,11 +15,78 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+<!-- Beginning of Nav Section and Great Idea bulb image -->
+  <body class="container">
+    <div>
+        <header class="container-header">
+          <nav class="nav-bar">
+             <a href="#">Services</a>
```

Comment:
a tag for Services should have some sort of local URL pointing to services - 
 ./myprojects/UI3/services.html